src/components/10-atoms/carousel/unit.test.js

```diff
@@ -40,23 +40,127 @@ describe('AXACarousel', () => {
   });
 
   describe('allgemeine private methods', () => {
-    test('_getSlides() should call querySelector(), assignedNodes() and filter()', () => {
-      const mockedFilter = jest.fn();
-      const mockedQuerySelector = jest.fn(() => {
-        return {
-          assignedNodes: () => {
-            return { filter: mockedFilter };
-          },
-        };
+    test('_getSlides() should get the elements and filter them (with ShadowRoot)', () => {
+      const mockedFilter = jest.fn(() => {
+        return [];
       });
+
+      const mockedShadowRoot = {
+        querySelector: () => {
+          return {
+            assignedNodes: () => {
+              return {
+                filter: mockedFilter,
+              };
+            },
+          };
+        },
+      };
+
       const mockedThis = {
-        querySelector: mockedQuerySelector,
+        querySelector: () => {},
+        shadowRoot: mockedShadowRoot,
+      };
+
+      AXACarousel.prototype._getSlides(mockedThis);
+
+      expect(mockedFilter).toHaveBeenCalled();
+    });
+
+    test('_getSlides() should get the elements and filter them (withOUT ShadowRoot)', () => {
```

Comment:
Why is withOUT written with uppercase?

Reply:
To see that it is different to the test "with ShadowRoot"

Reply:
To see that it is different to the test "with ShadowRoot"